### PR TITLE
fix(Popover): split renders

### PR DESCRIPTION
**Pull request by mainframev**
[Request in plz-orbit.
](https://skypicker.slack.com/archives/CAMS40F7B/p1618235742201200 )
[ORBIT-1721](https://jira.kiwi.com/browse/ORBIT-1721)
 Orbit.kiwi: https://orbit-docs-fix-popover-renders.surge.sh
 Storybook: https://orbit-fix-popover-renders.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/Popover/components/ContentWrapperMobile.js [hunk 1]
```diff
@@ -25,23 +21,17 @@ const mobileTop = theme => theme.orbit.spaceXLarge;
...
 
 const StyledContentWrapper = styled.div`
-  overflow: auto;
-  border-top-left-radius: 12px;
-  border-top-right-radius: 12px;
-  position: absolute;
-  left: 0;
-  width: 100%;
-  background-color: ${({ theme }) => theme.orbit.paletteWhite};
-  max-height: ${({ windowHeight, actionsHeight }) => `${windowHeight - actionsHeight - 32}px`};
```
Comment (silvenon) [inline packages/orbit-components/src/Popover/components/ContentWrapperMobile.js:35]: I can't find where this `windowHeight` was coming from before you started computing it inside 😄  Was it being passed from outside somehow…?

**Review by silvenon — CHANGES_REQUESTED** [review]
I checked this locally and it works 👍  The code seems straightforward, I would only suggest trying to add a unit test, as the Jira ticket description says as well, you can add a separate test file so that you can mock `useMediaQuery` differently. I was thinking of adding a story alternatively in case there's no way to test it with unit tests, but it would be better to use Cypress tests in that case, which we don't have yet.

File: packages/orbit-components/src/Popover/components/ContentWrapperDesktop.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    content,
+    fixed,
+    scrollableParent,
+    children,
+  });
+
+  const verticalPosition = calculateVerticalPosition(position[0], dimensions);
+  const horizontalPosition = calculateHorizontalPosition(position[1], dimensions);
+  const [actionsDimensions, setActionsDimensions] = React.useState(0);
+  const windowHeight = typeof window !== "undefined" ? window.innerHeight : 0;
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/src/Popover/components/ContentWrapperDesktop.js:139]: that's not the part of the pull request, i just moved the content, it was there, the aim of this PR is to fix the issue in the descriptionon

**Comment by mainframev** [comment]
Closed. Replaced by #2923

**Pull request closed without merging**